test/support/utils_invoke.h

```diff
@@ -157,6 +157,17 @@ inline void unsupported_types_notifier(const sycl::device& device)
     }
 }
 
+template<typename KernelName>
+auto
+create_test_policy(sycl::queue _queue)
```

Comment:
Are we able to  use already existing `make_new_policy` from `TestUtils` namespace?

Reply:
I looked at that function, but it requires a policy to already exist, and just creates one from it.  The code which originally creates the policy was embedded within `TestUtils::invoke_on_all_hetero_policies`, but now it lives here.
We could consolidate these, and provide an overload for a queue perhaps.  Would you prefer that?

Reply:
Up to you.

Reply:
I've added an overload of `make_new_policy` for queue inputs, and relocated them so they can live side by side.

Reply:
ok